test/moonwall.config.json

```diff
@@ -41,6 +41,7 @@
     {
       "name": "zombie_moonbeam_rpc",
       "testFileDir": ["suites/zombie"],
+      "multiThreads": false,
```

Comment:
Why to you need to make zombitest single thread? What is the relation with tracing tests?

Reply:
This was just experimentation, I suspect the test failures are not deterministic because we spawn multiple threads and sometimes it causes issues.